pages/components/toast/toast.css

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.toast-wrapper {
+  background: rgb(50, 50, 50);
+  padding: 1rem;
+  border-radius: 12px;
+  width: 100%;
+  max-width: 700px;
+}
+
+.toast-text {
+  color: var(--white-color);
+  font-weight: lighter;
+}
+
+.toast-btn {
+  position: relative;
+  background: transparent;
```

Comment:
Why position relative needed? Just use flex properties right. Mine worked that way. Just a suggestion

Reply:
Okay will try that.